src/applications/financial-status-report/pages/income/spouse/additionalIncome/enhancedIndex.js

```diff
@@ -1,20 +1,20 @@
 import SpouseAdditionalIncomeCheckList from '../../../../components/SpouseAdditionalIncomeCheckList';
 
 export const uiSchema = {
-  'ui:title': "Your spouse's other income",
-  additionalIncomeChecklist: {
-    'ui:title': 'Select any additional income your spouse receives:',
-    'ui:widget': SpouseAdditionalIncomeCheckList,
-    'ui:options': {
-      hideOnReview: true,
-    },
+  'ui:title': 'Monthly housing expenses',
+  'ui:field': SpouseAdditionalIncomeCheckList,
+  'ui:options': {
+    hideOnReview: true,
+  },
+  spouseAdditionalIncomeCheckList: {
+    'ui:title': 'spouseAdditionalIncomeCheckList',
   },
 };
 
 export const schema = {
   type: 'object',
   properties: {
-    additionalIncomeChecklist: {
+    spouseAdditionalIncomeCheckList: {
```

Comment:
I'm not 100% sure we can change the `additionalIncomeChecklist` to `spouseAdditionalIncomeChecklist` in this section lets loop in @Dr-Pongo  to verify if it's ok of it it would require additional changes with validation or updates to the form config